### DISCUSSION TITLE
Documenting ODF connections

### DISCUSSION THREAD

**Discussion by salomaaj** [netbox-community/netbox#6141] [Ideas]
I'm having trouble documenting ODF connections between sites / racks at the moment. Yes, the ports that are used are documented, but the ones that are not connected are not documented. Hard to keep track how many free pairs I have between X & Y especially when not all cables carry same amount of fibres.

Others with same problem? How do you get around the issue?

**Comment by candlerb** [comment]
Netbox 2.11 (currently in beta) will let you mark ports as in use, without actually connecting them to a cable.

Any port which is not connected is free.  So you could then write a custom report to list, for each ODF, the number of ports used and free - essentially just count all the FrontPorts and group them by device.
